ui/components/NodeInputButton.tsx

```diff
@@ -20,17 +24,45 @@ export const NodeInputButton: FC<NodeInputProps> = ({
   disabled,
   dispatchSubmit,
 }) => {
-  const handleClick = (e: MouseEvent | FormEvent) => {
-    const onClick = attributes.onclick;
+  const onClick = attributes.onclick ?? "";
+
+  // automatically start the webauthn login
+  const startWebAuthnLogin = () => {
+    const webauthnWindow = window as unknown as WebauthnWindow;
+    const loginParams = getWebAuthnPayload(onClick);
+
+    if (webauthnWindow.__oryStartedLogin) {
+      return;
+    }
+    webauthnWindow.__oryStartedLogin = true;
 
-    // webauthn registration has a custom handler in webauthn.js to be called
+    // retry login until the webauthn.js script is finished loading
+    // and the __oryWebAuthnLogin function is available
+    const triggerWebauthnLogin = (count: number) =>
+      setTimeout(() => {
+        if ("__oryWebAuthnLogin" in webauthnWindow) {
+          webauthnWindow.__oryWebAuthnLogin(loginParams);
+        } else if (count < 100) {
+          triggerWebauthnLogin(count + 1);
```

Comment:
question(possible-issue): this waits for the login func to be available, but my question is, since not all browser seem to support tail call optimization (ECMAScript 6), are we sure we don't risk any stack issues in case the webauthn.js script is slow to load or doesn't load at all (like hydra is not available or something) ?

Reply:
We can reduce the max nesting level, but I think 100 should be safe even with legacy browsers.